### PR TITLE
Uninstall cluster-logging from OCP leaves behind cluster-logging operator and elastic-search operator

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1783620

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge @jcantrill Do we want to add steps to the `Uninstall cluster logging` topic to remove all remnants of cluster logging and ES?

**Comment by mburke5678** [comment]
@sichvoge @jcantrill Do we want to add steps to the Uninstall cluster logging topic to remove all remnants of cluster logging and ES?

**Comment by sichvoge** [comment]
I think we should definitely have something similar to https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/installing-ossm.html#removing-red-hat-openshift-service-mesh and the subsequent section, maybe even provide a small script that removes all remaining objects related to Logging (e.g. https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/installing-ossm.html#ossm-remove-cleanup_installing-ossm).

@mburke5678 @jcantrill WDYT?

File: modules/cluster-logging-uninstall.adoc [hunk 1]
```diff
@@ -29,3 +29,48 @@ after the Operator instances are deleted:
...
+. Optional: Use the CLI to remove the Cluster Logging and Elasticsearch operator groups.
++
+[source,terminal]
+----
+$ oc delete og cluster-logging -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc delete og openshift-operators-redhat -n openshift-operators-redhat
```
Comment (jcantrill) [inline modules/cluster-logging-uninstall.adoc:62]: We should be careful about this recommendation as its possible other RH global operators could be here

File: modules/cluster-logging-uninstall.adoc [hunk 2]
```diff
@@ -29,3 +29,48 @@ after the Operator instances are deleted:
...
+
+.. On the *Custom Resource Definitions* page, click *Elasticsearch*.
+
+.. On the *Custom Resource Definition Details* page, select *Delete Custom Resource Definition* from the *Actions* menu.
+
+. Optional: Use the CLI to remove the Cluster Logging and Elasticsearch operator groups.
++
+[source,terminal]
+----
+$ oc delete og cluster-logging -n openshift-logging
```
Comment (jcantrill) [inline modules/cluster-logging-uninstall.adoc:57]: By doing this you effectively will remove any CL resources that live in the namespace which means you would not have to do any of the steps listed above specific to the namespace

File: modules/cluster-logging-uninstall.adoc [hunk 3]
```diff
@@ -29,3 +29,48 @@ after the Operator instances are deleted:
...
+. Optional: Use the CLI to remove the Cluster Logging and Elasticsearch projects.
++
+[source,terminal]
+----
+$ oc delete project cluster-logging
+----
++
+[source,terminal]
+----
+$ oc delete project openshift-operators-redhat
```
Comment (jcantrill) [inline modules/cluster-logging-uninstall.adoc:74]: my previous comment regarding deleting the namespace applies here

**Comment by mburke5678** [comment]
@anpingli PTAL

File: modules/cluster-logging-uninstall.adoc [hunk 4]
```diff
@@ -15,17 +15,96 @@ You can remove cluster logging from your cluster.
...
+. Use the following command to remove the Cluster Logging CR.
 +
 [source,terminal]
 ----
 $ oc delete clusterlogging instance -n openshift-logging
 ----
 
-. Use the following command to remove the Persistent Volume Claims that remain
-after the Operator instances are deleted:
+. Optional: Use the following command to remove any Cluster Logging Persistent Volume Claims:
```
Comment (anpingli) [inline modules/cluster-logging-uninstall.adoc:25]: Delete only pvc for clusterlogging
oc delete pvc --selector=logging-cluster=elasticsearch -n openshift-logging

File: modules/cluster-logging-uninstall.adoc [hunk 5]
```diff
@@ -15,17 +15,96 @@ You can remove cluster logging from your cluster.
...
 +
```
Comment (anpingli) [inline modules/cluster-logging-uninstall.adoc:26]: For 4.6+,[optional] need to delete ClusterLogForwarder CR, if you had enabled ClusterLogForwarder.
oc delete ClusterLogForwarder instance -n openshift-logging

File: modules/cluster-logging-uninstall.adoc [hunk 6]
```diff
@@ -15,17 +15,54 @@ You can remove cluster logging from your cluster.
...
-$ oc delete pvc --all -n openshift-logging
-----
+[IMPORTANT]
+====
+Do not delete the `openshift-operators-redhat` project if other global operators are installed in this namespace.
+====
+
+.. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
+
+. Optional: Remove any Cluster Logging Persistent Volume Claims (PVC):
```
Comment (ahardin-rh) [inline modules/cluster-logging-uninstall.adoc:63]: s/Persistent Volume Claims/persistent volume claims

**Comment by ahardin-rh** [comment]
@mburke5678 Just one suggestion from me. Otherwise, looks great!

**Pull request merged**